src/registrar/assets/sass/_theme/_base.scss

```diff
@@ -223,3 +223,9 @@ abbr[title] {
 .padding--8-8-9 {
   padding: 8px 8px 9px !important;
 }
+
+.usa-icon.usa-icon--big {
```

Comment:
You probably only need .usa-icon--big as a selector, you probably want to put this in the _buttons file